protocol-units/da/m1/util/Cargo.toml

```diff
@@ -43,6 +43,7 @@ zstd = { workspace = true }
 bcs = { workspace = true }
 ecdsa = { workspace = true, features = [ "signing", "verifying", "der"] }
 k256 = { workspace = true }
+rand = { version = "0.8.5" }
```

Comment:
Use the workspace version?

Reply:
It conflicts with Aptos. Aptos is on `0.7` ecdsa is on `0.8`. I will leave a comment.